### ISSUE TITLE
Submission always failing - even before scoring program

### ISSUE THREAD

**Issue by michaal94** [codalab/codalab-competitions#3499]
Hey Codalab Team,

I have an issue with one of my competitions. Namely, all the submissions fail immediately after the status switches to running.

When I submit anything to this competition: https://codalab.lisn.upsaclay.fr/competitions/19222 the status switches to submitting, then 'running', and almost immediately 'failed'.

I have a second competition https://codalab.lisn.upsaclay.fr/competitions/19223 which uses the exact same scoring program. For this competition, submissions run without issues. 

I tested the first competition by switching the reference data to be the same as in the second (the scoring program is already the same) and uploaded the same file; hence, the input and ground truth were the same in both competitions. However, the submission works on the second challenge (19223) and fails on the first one (19222). I also tested with a very small submission size (dummy images) and there is no change. I can add that in the 19223 competition which is working, when the status changes to 'running' the submission size gradually increases to the file size value, whereas in 19222 which fails, it stops roughly at 2700 bytes as the submission size.

I would be grateful for any help resolving that.

**Comment by Didayolo** [comment]
Hi @michaal94, let me continue investigating here.

I see only one submission on each of the competition. Is it normal? Where are the failed submissions?

**Comment by michaal94** [comment]
Hey, currently I managed to get it to work. I uploaded the scoring program and reference data for the second time, I started changing the options of the competition (e.g. docker image) - and after this, it was still not working. Finally, I decided to revert everything back to the original state and it started working again. 

(After it started working, I removed all submissions apart from one)

So currently it is working, thank you.
